aws-cpp-sdk-transfer/include/aws/transfer/TransferHandle.h

```diff
@@ -154,6 +154,14 @@ namespace Aws
              */
             TransferHandle(const Aws::String& bucketName, const Aws::String& keyName, CreateDownloadStreamCallback createDownloadStreamFn, const Aws::String& targetFilePath = "");
 
+            /**
+             * Alternate DOWNLOAD constructor
+             */
+            TransferHandle(const Aws::String& bucketName, const Aws::String& keyName, 
```

Comment:
`offset` here is not clear of what kind of `offset`, I'd use `fileOffset` and `downloadBytes`. Also document that `fileOffset` currently only used for download.